### ISSUE TITLE
LCOM4 is not counted

### ISSUE THREAD

**Issue by lyriccoder** [yegor256/jpeek#452]
[ChineseRemainderTheorem.zip](https://github.com/yegor256/jpeek/files/4494048/ChineseRemainderTheorem.zip)

I downloaded the last version https://repo1.maven.org/maven2/org/jpeek/jpeek/0.30.9/jpeek-0.30.9-jar-with-dependencies.jar

Then I compiled Java file and got byte-code for each class. But when I ran  jpeek from the command line, I did not see LCOM4 metrics.

`java -jar jpeek-jar-with-dependencies.jar --sources . --target ./jpeek`

Could you please tell me, how is it possible to see LCOM4 metric?

![image](https://user-images.githubusercontent.com/20803206/79590879-6bedd600-80e0-11ea-9850-933b975f78df.png)

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) please, pay attention to this issue

**Comment by paulodamaso** [comment]
@lyriccoder Seems like its `Report` is not being generated

**Comment by paulodamaso** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/yegor256/jpeek/issues/452#issuecomment-616476404))

@paulodamaso Job [#452](https://github.com/yegor256/jpeek/issues/452) doesn't exist, can't set role

**Comment by lyriccoder** [comment]
I have added examples of java classes in the archive. Could you please check? I want to compute LCOM4, but I would like to know what is happening and why the report was not generated

**Comment by paulodamaso** [comment]
@0crat assign me

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/yegor256/jpeek/issues/452#issuecomment-617210134))

@paulodamaso The job [#452](https://github.com/yegor256/jpeek/issues/452) assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/e83cbc98-f6c1-45f2-b15f-a73e640c8508); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by paulodamaso** [comment]
@lyriccoder `LCOM4` report was not being added to the report stack; because of this, jPeek wasn't generating it even if we passed asked just for this report. I've added `LCOM4` report generation in PR #458; after its merge, you'll be able to generate it by using the `--metrics` parameter, like:
```
java -jar target/jpeek-jar-with-dependencies.jar --sources . --target ./jpeek --overwrite --metrics "LCOM4"
```

**Comment by lyriccoder** [comment]
oh, thank you very much @paulodamaso

**Comment by 0pdd** [comment]
@lyriccoder 2 puzzles [#460](https://github.com/yegor256/jpeek/issues/460), [#461](https://github.com/yegor256/jpeek/issues/461) are still not solved.

**Comment by 0crat** [comment]
Job [#452](https://github.com/yegor256/jpeek/issues/452) is not in the agenda of @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), can't set title

**Comment by 0pdd** [comment]
@lyriccoder 2 puzzles [#494](https://github.com/cqfn/jpeek/issues/494), [#506](https://github.com/cqfn/jpeek/issues/506) are still not solved.

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/cqfn/jpeek/issues/452#issuecomment-617210134))

@paulodamaso Job [#452](https://github.com/cqfn/jpeek/issues/452) is closed, can't start order

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/cqfn/jpeek/issues/452#issuecomment-616476404))

@paulodamaso @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) not enough funds available in the project, can't set budget of job [#452](https://github.com/cqfn/jpeek/issues/452), see [§21](http://www.zerocracy.com/policy.html#21); @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) will get no money on completion; in order to fix that, add funds to the project and assign the job again

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) resigned from [#452](https://github.com/cqfn/jpeek/issues/452), since the job is not in scope anymore

**Comment by 0pdd** [comment]
@lyriccoder all 2 puzzles are solved here: [#494](https://github.com/cqfn/jpeek/issues/494), [#506](https://github.com/cqfn/jpeek/issues/506).
